### PR TITLE
Assessment item path fix

### PR DESCRIPTION
**Pull request by benjaoming**
#3898

### CODE REVIEW

**Comment by benjaoming** [comment]
@MCGallaspy @rtibbles I'm going to do something for aforementioned issue. Would you have comments on this?

File: kalite/distributed/management/commands/setup.py [hunk 1]
```diff
@@ -287,9 +287,33 @@ def handle(self, *args, **options):
...
         # download assessment items
         # This can take a long time and lead to Travis stalling. None of this is required for tests.
-        if options['force-assessment-item-dl']:
+
+        # Outdated location of assessment items
+        # TODO(benjaoming) for 0.15, remove this
+        writable_assessment_items = os.access(settings.KHAN_ASSESSMENT_ITEM_ROOT, os.W_OK)
+        
+        def _move_to_new_location(old, new):
+            if not writable_assessment_items
```
Comment (MCGallaspy) [inline kalite/distributed/management/commands/setup.py]: Syntax error?

**Comment by MCGallaspy** [comment]
I'm not sure I fully understand the need for this. It looks like you're trying to ensure that the assessment items files are in a user-writable location, yes? But I think the intention is that assessment items are not user-writable. My understanding is that they're not committed to the repo like the other khan data in order to not inflate the size, but that they are essentially "static" data like the python dependencies.

I guess I'm still not entirely clear; for uncommitted but write-able files ("user data", like data.sqlite), it's not appropriate for the files to be mixed up with the source. But this data (`assessment.sqlite`) is not meant to be write-able; once it is downloaded or otherwise resolved at installation time, can't we keep it bundled up with the source?

@jamalex's input might be helpful here.

**Comment by MCGallaspy** [comment]
Basically my point is that we should treat this file like `data/khan/topics.json`, except that it's created/downloaded at installation time.

**Comment by benjaoming** [comment]
@MCGallaspy Motivation is explained in the issue #3898

**Pull request closed without merging**